tests/13-basic-attrs.c

```diff
@@ -123,6 +123,20 @@ int main(int argc, char *argv[])
 		goto out;
 	}
 
+	expected = 1;
+	rc = seccomp_attr_set(ctx, SCMP_FLTATR_NEW_LISTENER, expected);
```

Comment:
See my previous comments about this test as well as the missing Python equivalent.